### PR TITLE
Init snake_cases_fixes

### PR DESCRIPTION
**Pull request by EdVeralli**
WIP - Snake_Cases fixes in function names

### CODE REVIEW

File: src/main.leo [hunk 1]
```diff
@@ -189,12 +188,12 @@ function test_balanceof(owner_balance: Balance) -> bool {
 	}
 
 	// Test	
-	let test_balanceoff_bool: bool = false;
+	let test_balanceof_bool: bool = false;
```
Comment (Loco1982) [inline src/main.leo:191]:
```suggestion
	let test_balance_of_bool: bool = false;
```

File: src/main.leo [hunk 2]
```diff
@@ -189,12 +188,12 @@ function test_balanceof(owner_balance: Balance) -> bool {
...
 	if return_balance == 269u64{
-		test_balanceoff_bool = true;
+		test_balanceof_bool = true;
```
Comment (Loco1982) [inline src/main.leo:193]:
```suggestion
		test_balance_of_bool = true;
```

File: src/main.leo [hunk 3]
```diff
@@ -189,12 +188,12 @@ function test_balanceof(owner_balance: Balance) -> bool {
...
 	}
 	
-	return test_balanceoff_bool;
+	return test_balanceof_bool;
```
Comment (Loco1982) [inline src/main.leo:196]:
```suggestion
	return test_balance_of_bool;
```

**Review by Loco1982 — CHANGES_REQUESTED** [review]
Also there's a typo in the variable called "minimun", it should be "minimum". It's called a couple of times, particularly in the tests

File: src/main.leo (hunk 2 above)
Comment (EdVeralli) [reply src/main.leo:193]: Yes, you are right one more time..

**Comment by EdVeralli** [comment]
> Also there's a typo in the variable called "minimun", it should be "minimum". It's called a couple of times, particularly in the tests

I´ll resolve it.

**Review by Loco1982 — APPROVED** [review]
LGTM

**Review by pablodeymo — APPROVED** [review]
LGTM

**Pull request merged**